cypress/integration/login.js

```diff
@@ -1,29 +1,121 @@
-describe('Sourcebooks login', function() {
-
-    it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
-    })
-
-    it('Should be able to login with role User', function () {
-
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Team Lead"]').click();
-        cy.get('[type="submit"]').click();
-
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 2);
-        let today = new Date();
-        cy.get('.calendar--today').find('.calendar__date').contains(today.getDate());
-    })
-})
+describe('Sourcebooks login', function() {
```

Comment:
You only need one "describe" block that would have many "it" blocks.
You should move all "it" blocks to one "describe"